frontend/wasm/src/api.rs

```diff
@@ -17,6 +17,7 @@ use editor::Color;
 use editor::Editor;
 use editor::LayerId;
 
+use graphene::Operation;
```

Comment:
Please move one line up, into the previous section (which is for first-party crates).